bootstrap/app.php

```diff
@@ -16,6 +16,10 @@
                 ->prefix('api/v1')
                 ->group(base_path('routes/api-v1.php'));
 
+            Route::middleware('api')
+                ->prefix('api/v1/admin')
```

Comment:
@dhrumin-7span The admin API file name shoud be admin-v1.php

Reply:
Changed admin api file name.